src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1488,6 +1489,11 @@ void AutomationInstrumentClipView::editPadAction(bool state, uint8_t yDisplay, u
 
 			uiNeedsRendering(this);
 		}
+
```

Comment:
if you've let go of all pads, reset the lastPadSelectedKnobPos to the "unselected value" and re-draw the parameter name currently being edited